### PR TITLE
ID localization for administer cluster - network policy - calico

### PR DESCRIPTION
**Pull request by ariscahyadi**
This PR addressing `/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy` as mentioned here #22296

### CODE REVIEW

File: content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    ```shell
+    gcloud container clusters create [CLUSTER_NAME] --enable-network-policy
+    ```
+
+    **Contoh**
+    ```shell
+    gcloud container clusters create my-calico-cluster --enable-network-policy
+    ```
+
+2.  Untuk memverifikasi penyebaran, gunakanlah perintah berikut ini.
```
Comment (danninov) [inline content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md:33]:
```suggestion
2.  Untuk memverifikasi penggelaran, gunakanlah perintah berikut ini.
```

File: content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    ```shell
+    kubectl get pods --namespace=kube-system
+    ```
+
+    Pod Calico dimulai dengan kata `calico`. Periksa untuk memastikan statusnya `Running`.
+
+## Membuat klaster lokal Calico dengan kubeadm {#membuat-klaster-calico-dengan-kubeadm}
+
+Untuk mendapatkan satu klaster Calico dengan hos tunggal dalam waktu lima belas menit dengan menggunakan kubeadm, silahkan merujuk pada
```
Comment (danninov) [inline content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md:43]:
```suggestion
Untuk mendapatkan satu klaster Calico dengan hos tunggal dalam waktu lima belas menit dengan menggunakan kubeadm, silakan merujuk pada
```

File: content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+weight: 10
+---
+
+<!-- overview -->
+Laman ini menunjukkan beberapa cara cepat untuk membuat klaster Calico dalam Kubernetes.
+
+
+## {{% heading "prerequisites" %}}
+
+Putuskan apakah kamu ingin menyebarkan (_deploy_) sebuah [_cloud_](#membuat-klaster-calico-menggunakan-google-kubernetes-engine-gke) atau klaster [lokal](#membuat-klaster-calico-dengan-kubeadm).
```
[the remark this answers is not in the document]
Comment (ariscahyadi) [reply content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md:13]: I am agree with "menggelar".

File: content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
+---
+title: Menggunakan Calico untuk NetworkPolicy
+content_type: task
+weight: 10
+---
+
+<!-- overview -->
+Laman ini menunjukkan beberapa cara cepat untuk membuat klaster Calico dalam Kubernetes.
```
Comment (phanama) [inline content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md:8]:
```suggestion
Laman ini menunjukkan beberapa cara cepat untuk membuat klaster Calico pada Kubernetes.
```

File: content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+
+## {{% heading "prerequisites" %}}
+
+Putuskan apakah kamu ingin menggelar (_deploy_) sebuah [_cloud_](#membuat-klaster-calico-menggunakan-google-kubernetes-engine-gke) atau klaster [lokal](#membuat-klaster-calico-dengan-kubeadm).
```
Comment (phanama) [inline content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md:13]:
```suggestion
Putuskan apakah kamu ingin menggelar (_deploy_) sebuah klaster di [_cloud_](#membuat-klaster-calico-menggunakan-google-kubernetes-engine-gke) atau di [lokal](#membuat-klaster-calico-dengan-kubeadm).
```

File: content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    gcloud container clusters create my-calico-cluster --enable-network-policy
+    ```
+
+2.  Untuk memverifikasi penggelaran, gunakanlah perintah berikut ini.
+
+    ```shell
+    kubectl get pods --namespace=kube-system
+    ```
+
+    Pod Calico dimulai dengan kata `calico`. Periksa untuk memastikan statusnya `Running`.
```
Comment (phanama) [inline content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md:39]:
```suggestion
    Pod Calico dimulai dengan kata `calico`. Periksa untuk memastikan bahwa statusnya `Running`.
```

File: content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+## Membuat klaster lokal Calico dengan kubeadm {#membuat-klaster-calico-dengan-kubeadm}
+
+Untuk mendapatkan satu klaster Calico dengan hos tunggal dalam waktu lima belas menit dengan menggunakan kubeadm, silakan merujuk pada
```
Comment (phanama) [inline content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md:43]:
```suggestion
Untuk membuat satu klaster Calico dengan hos tunggal dalam waktu lima belas menit dengan menggunakan kubeadm, silakan merujuk pada
```

File: content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+[Memulai cepat Calico](https://docs.projectcalico.org/latest/getting-started/kubernetes/).
+
+
+## {{% heading "whatsnext" %}}
+
+Setelah klaster kamu berjalan, kamu dapat mengikuti [mendeklarasikan kebijakan jaringan](/id/docs/tasks/administer-cluster/declare-network-policy/) untuk mencoba NetworkPolicy dari Kubernetes.
```
Comment (phanama) [inline content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md:50]:
```suggestion
Setelah klaster kamu berjalan, kamu dapat mengikuti [Mendeklarasikan Kebijakan Jaringan](/id/docs/tasks/administer-cluster/declare-network-policy/) untuk mencoba NetworkPolicy Kubernetes.
```

**Comment by ariscahyadi** [comment]
> Thanks for the translation! I've left some reviews for clarity @ariscahyadi

Hi @phanama, the comments have been addressed. Please check!

File: content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+weight: 10
+---
+
+<!-- overview -->
+Laman ini menunjukkan beberapa cara cepat untuk membuat klaster Calico pada Kubernetes.
+
+
+## {{% heading "prerequisites" %}}
+
+Putuskan apakah kamu ingin menggelar (_deploy_) sebuah klaster di [_cloud_](#membuat-klaster-calico-menggunakan-google-kubernetes-engine-gke) atau di [lokal](#membuat-klaster-calico-dengan-kubeadm).
```
Comment (girikuncoro) [inline content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md:13]: nit: cloud doesn't need italicize according to RSNI
```suggestion
Putuskan apakah kamu ingin menggelar (_deploy_) sebuah klaster di [cloud](#membuat-klaster-calico-menggunakan-google-kubernetes-engine-gke) atau di [lokal](#membuat-klaster-calico-dengan-kubeadm).
```

File: content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+
+
+<!-- steps -->
+## Membuat klaster Calico dengan menggunakan _Google Kubernetes Engine_ (GKE) {#membuat-klaster-calico-menggunakan-google-kubernetes-engine-gke}
```
Comment (girikuncoro) [inline content/id/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md:17]: nit: I think GKE doesn't need italic
```suggestion
## Membuat klaster Calico dengan menggunakan Google Kubernetes Engine (GKE) {#membuat-klaster-calico-menggunakan-google-kubernetes-engine-gke}
```

**Pull request merged**